TRANSFORMERS/CALCULUS/DOUBLE_INDEFINITE_INTEGRAL/DOUBLE_INDEFINITE_INTEGRAL.py

```diff
@@ -2,6 +2,10 @@
 import numpy as np
 
 
+def contains_only_numbers(column):
+    return all(isinstance(value, (np.int_, np.float_, int, float)) for value in column)
```

Comment:
Pylance raises an error for this line with the following message: "Second argument to 'isinstance' must be a class or tuple of classes. TypeVar or generic type with type arguments not allowed."